charts/kube-prometheus-stack/values.yaml

```diff
@@ -69,6 +69,9 @@ defaultRules:
   ## Additional labels for PrometheusRule alerts
   additionalRuleLabels: {}
 
+  ## Prefix for runbook URLs
+  runbookUrl: ""
```

Comment:
As it looks like, the runbookUrl needs to end with a trailing slash. Should we add this to the comment above?

Reply:
yeah - I think we should.

Reply:
The new default runbookUrl needs to end in a trailing slash. If you override it there are a lot of possibilities some of which would not need to end in a trailing slash. The following would both build perfectly valid URLs:

* `https://example.com/path/to/endpoint?alert=`
* `https://example.com/path/to/html-page#`